tests/all/volumes.t

```diff
@@ -100,6 +102,33 @@ USE_NODE nodes[0]
 TEST "kadalu volume create DEV/vol5 server1:/exports/vol5/s1 server2:/exports/vol5/s2 server3:/exports/vol5/s3 --auto-create-pool --auto-add-nodes"
 TEST "kadalu volume stop DEV/vol5 --mode=script"
 TEST "kadalu volume delete DEV/vol5 --mode=script"
+
+# Tests for volume re-use usecase
+def create_volume_and_get_id(cmd)
+    TEST "echo create command: #{cmd}"
```

Comment:
If you want to put some print messages then no need to use `TEST` and echo. It starts child process to call `echo`. Instead just use `puts`.

```ruby
puts "create command: #{cmd}"
out = TEST cmd
puts out
last_line = out.strip.split("\n")[-1]
vol_id = last_line.nil? ? "" : last_line.split(":")[-1].strip
puts last_line
puts vol_id

vol_id
```

Also note last line in the function is returned automatically without adding `return`. For example below two examples are same

```ruby
def create_volume_and_get_id(cmd)
     ...
     return vol_id
end
```

```ruby
```ruby
def create_volume_and_get_id(cmd)
     ...
     vol_id
end
```

in your example, last line is the output of `echo #{last_line}` instead of returning the output of Line 112.

Reply:
returned `vol_id`